doc/faq.md

```diff
@@ -215,6 +215,14 @@ For `.vue` single file components it uses the `vue-template-compiler`
 
 - which will be in your module dependencies if you're developing with Vue).
 
+### Q: Does this work with Svelte as well?
+
+**A**: Yes.
+
+For `.svelte` single file components it uses the `svelte2tsx`
+
+- you might have to install it as peer dependency `npm i -D svelte2tsx`.
```

Comment:
```suggestion
- you might have to install it as development dependency if you don't have it already: `npm i -D svelte2tsx`.
```

Reply:
no longer relevant due to https://github.com/sverweij/dependency-cruiser/pull/393#discussion_r523383294